### PR TITLE
Added location logging to user dashboard

### PR DESCRIPTION
**Pull request by christinachance**

### CODE REVIEW

**Review by rivera-fernando — APPROVED** [review]

File: project/src/main/java/com/google/sps/servlets/LogLocation.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        locationEntity.setProperty("lng", longitude);
+        locationEntity.setProperty("lat", latitude);
+        locationEntity.setProperty("email", email);
+        locationEntity.setProperty("date", dateStamp);
+
+        datastore.put(locationEntity);
+        response.sendRedirect("/dashboard");
+
+    }
+}
```
Comment (rivera-fernando) [inline project/src/main/java/com/google/sps/servlets/LogLocation.java]: newline end of file

Comment (christinachance) [reply project/src/main/java/com/google/sps/servlets/LogLocation.java]: done.

**Review by Tcintra — APPROVED** [review]

File: project/src/main/java/com/google/sps/classes/User.java [hunk 2]
```diff
@@ -12,11 +12,11 @@
   private final String school;
   private String phone;
   private String metric;
-  private Boolean admin;
+  private Boolean is_admin;
   private String imageKey;
 
   /*User class is instatiated when a user logins so all their info is accesible without the datatsore being queried each time*/
-  public User(long userId, String name, String email, String password, String birthdate, long studentId, String sex, String school, String phone, String metric, Boolean admin, String imageKey) {
+  public User(long userId, String name, String email, String password, String birthdate, long studentId, String sex, String school, String phone, String metric, Boolean is_admin, String imageKey) {
```
Comment (serzhk) [inline project/src/main/java/com/google/sps/classes/User.java]: Could you keep line shorter? Usually at Google we are using 80 char line

**Review by serzhk — APPROVED** [review]

File: project/src/main/java/com/google/sps/classes/User.java (hunk 2 above)
Comment (christinachance) [reply project/src/main/java/com/google/sps/classes/User.java]: done.

**Pull request merged**